src/components/products/Products.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from "react";
+import { BsHeart } from "react-icons/bs";
+import { BsStarFill } from "react-icons/bs";
+import DiscountedPrice from "../../utils/DiscountedPrice";
+import "./products.css";
+import axios from "axios";
+
+
+const Products = () => {
+  const [productsList, setProductsList] = useState([]);
+
+  const loadProducts = async () => {
+    try {
+      const response  = await axios.get("/api/products");
+       console.log(response.data);
+      setProductsList(response.data.products);
+    } catch {
+      console.log("caught error from products.jsx");
+      //todo: toast addition
+    }
+  };
+  useEffect(() => loadProducts(), []);
+
+  return (
+    <div className="child pd-products-layout">
+      <div className="grid-column-layout">
+        {productsList.map(
+          ({
+            title,
+            categoryName,
+            brand,
+            image,
+            tags,
+            price,
+            ratings,
+            stockQty,
+            discount,
+            comingSoon
+          }) => {
+            return (
+              <div className="card-container parent-container">
+                <div className="parent-image">
+                  <img className="image-container" src={image} alt={title} />
+                  <div className="tags-flex">
+                    {tags ? (
+                      tags === "new" ? (
+                        <div className="tags-badge child-tags bg-baseblue">
+                          New
+                        </div>
+                      ) : (
+                        <div className="tags-badge child-tags bg-baseorange">
+                          Trending
+                        </div>
+                      )
+                    ) : (
+                      <></>
+                    )}
```

Comment:
`):null }` this will also work, no need to write empty react fragment.
Alternatively you can use `&&`

Reply:
done